80_shell.Rmd

```diff
@@ -6,7 +6,7 @@ Even if you do most of your Git operations via a client, such as RStudio or GitK
 
 ## What is the shell?
 
-The `shell` (or `bash` or terminal) is a program on your computer whose job is to run other programs, rather than do calculations itself. The `shell` is a very old program and in a time before the mouse it was the only way to interact with a computer. It is still extremely popular among programmers because it is very fast, concise, and is particularly powerful at automating repetitive tasks.
+The `shell` (or `bash` or terminal or Command Prompt on Windows) is a program on your computer whose job is to run other programs, rather than do calculations itself. The `shell` is a very old program and in a time before the mouse it was the only way to interact with a computer. It is still extremely popular among programmers because it is very fast, concise, and is particularly powerful at automating repetitive tasks.
```

Comment:
So will you add some verbiage that makes it clear we're talking about bash? And if a windows user is getting puzzling results, that suggests they haven't installed git bash yet? And link to those install instructions?

@kevinushey tells me RStudio will launch cmd.exe if there's no other option (also seems to be related whether current project is a git repo?), so I don't think we can assume that ALL shells launched via RStudio are bash. Early on, while someone is still setting up, it is possible they will end up in cmd.exe.